packages/venia-concept/src/components/Checkout/braintreeDropin.js

```diff
@@ -86,7 +84,10 @@ class BraintreeDropin extends Component {
         );
     }
 
-    createDropinInstance = () => {
+    createDropinInstance = async () => {
+        // import the dropin API
+        const { default: dropin } = await import('braintree-web-drop-in');
+
```

Comment:
I'm pretty sure `React.lazy` only works with components, so we'll do this dynamic import manually.